### DISCUSSION TITLE
Cards get stuck after moving for a new column.

### DISCUSSION THREAD

**Discussion by oalexandrino** [kanboard/kanboard#5194] [General]
### Actual behaviour

After deploying, I realized that the the cards get stuck after moving for a new column.
It happens on both at my personal web server and on heroku service.
Actually, the cards are moved. I can check it out after manually refreshing the browser. But from the user experience perspective, the cards keeps loading and never stop loading.

### Expected behaviour

No slow after moving card between columns.

### Steps to reproduce

Just install it, and try it after creating and moving cards.

### Screenshots

It can be seen at the attached image.

### Configuration

- Application version: 1.2.14
- PHP version: 7.2.31
- PHP SAPI: litespeed
- HTTP Client: cURL
- OS version: Linux 3.10.0-962.3.2.lve1.5.35.el7.x86_64
- Database driver: sqlite
- Database version: 3.26.0
- Browser: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) 
![kanboard-stuck](https://user-images.githubusercontent.com/1805816/84810549-e2745a80-afe1-11ea-8f1d-fcb85bfda3e3.png)
Chrome/83.0.4103.97 Safari/537.36

**Comment by identei** [comment]
Hello, I am unable to replicate this so far. It might be that the network request happening when you move a card is failing and that's why it never stops showing the loading icon. Any chance you can open up the browser's developer tools and see if the request is failing? It is supposed to look like the screenshot I've attached.

![image](https://user-images.githubusercontent.com/25444175/89725728-a51fbe00-d9cf-11ea-833d-aec0403a4f1d.png)

**Comment by pamartinez666** [comment]
Hi, could you solve it? the same happens to me.

**Comment by oalexandrino** [comment]
> Hi, could you solve it? the same happens to me.

Hello! Try to do the following:

- disable the feature that sends e-mail when an activity is done. Check if the problem goes away. It may be the reason of the problem. If the problem keeps...follow the next item
- if you have installed any plugin you will need to remove one by one
- after you have removed the first one, try it. Check if the problem goes away..
- do it for each plugin that you have installed

**Comment by identei** [comment]
@oalexandrino does that mean you were able to figure out which plugin you were using that was causing it? Were you able to get it working? If so, we may be able to close this and open it in the specific plugin causing the problem.

**Comment by oalexandrino** [comment]
@SladeP sure! BUT I am not sure IF it was some plugin that created this undesired behavior. It is a supposition. What did I do? I removed all the plugins so that it stopped the issue. I had installed several plugins. Maybe the problem is in ONE of them, maybe not. I couldn't find the real reason.  Thus, it needs more investigation.

**Comment by identei** [comment]
@oalexandrino do you have that list of plugins you had installed? I can take a look and try to figure out which one.

**Comment by pamartinez666** [comment]
Hi, in my case the error was produced by the Bigboard plugin.

**Comment by identei** [comment]
@pamartinez666 thank you! Looks like Bigboard slows the requests down and replaces the AJAX call made when moving a card, which could be what made the loading icon appear for a long time. Bigboard makes the ajax calls take 3+ seconds as opposed to <1 without it. Might be worth opening an issue in their project to investigate speeding up the ajax controller they use.

**Comment by oalexandrino** [comment]
@SladeP SladeP I'm sorry for the delay, I haven't seen the updates.

Here you're:


Current plugins installed (today, there is no problem)

- Customizer
- Bigboard
- TaskAssignCategory
- CardPushDate
- KanboardCommentActions
- Group_assign
- TaskAssignDateToUndated
- Calendar
- Gantt
- TimeMachine
- Self-Registration
- Task2pdf
- StarredProjects
- Google Authentication

Plugins removed (at that time, there was a problem. Again, I am no sure if one of them is the root of the problem)

- ProjectInvitation
- Assign category when moving cards to specified column
- Assign date to undated cards
- Chat
- Card Push Date - and other actions
